src/main.py

```diff
@@ -13,20 +14,48 @@
 If an ARTag exists, the vehicle will become a follower vehicle.
 If an ARTag does not exist, the vehicle will become the leader vehicle.
 """
```

Comment:
This docu string should be located at the very top (with your name on it) since it's explaining what the file itself is doing instead of the function below the comment.